utils/telegram.py

```diff
@@ -66,6 +66,7 @@ def broadcast(self, message, filter_str=''):
             for conversation in conversations:
                 bot.send_chat_action(chat_id=conversation, action=ChatAction.TYPING)
                 bot.send_message(chat_id=conversation, text=chunk)
+                if sticker: bot.send_sticker(chat_id=conversation, text=chunk, sticker=sticker)
```

Comment:
1. Does this send `text=chunk` again? Or is this an ignored parameter?
2. This is inside a loop of chunks. If you ask Ada to send a message bigger than 1024 characters, she will repeatedly send the sticker in between the messages. This should be in an outer loop of its own, or you should switch the `for`s around.